feat: add xchain testnet and mainnet

dileepfrog opened the pull request.

Morhef commented on _data/chains/eip155-64002.json:
_data/chains/eip155-94524.json

Morhef reviewed (commented):
_data/chains/eip155-94524.json

ligi reviewed (changes requested):
you cannot delete chains - only mark them deprecated. This is important to prevent replay attacks

dileepfrog commented:
Added deprecation notice to old xchain listings - this unrelated project appears defunct as none of the provided URLs work. Added `deprecated` string where applicable to avoid naming collision. Please advise what else we can do to get this merged, thank you!

dileepfrog commented:
The build error is for the old chain that we flagged deprecated:

`Exception in thread "main" java.net.UnknownHostException: xchainscan.com: No address associated with hostname`

ligi reviewed (changes requested):
remove these URLS then

dileepfrog commented:
> remove these URLS then

Removed
